src/index.html

```diff
@@ -8,5 +8,40 @@
   </head>
   <body>
     <h1>Calendar</h1>
+    <div class="calender">
+      <div class="calender__field calender__field--month-length--29">
+        <div class="calender__cell calender__cell--start-day--sat"></div>
```

Comment:
```suggestion
    <div class="calendar calendar--29 calendar--sat">
      <div class="calendar__cell"></div>
```
calendar пишется через a
тебе не нужен блок calendar__field
все классы-модификаторы должны быть у самого calendar, а не у calendar__cell